com.microsoft.mrtk.graphicstools.unity/Runtime/Shaders/GraphicsToolsStandardProgram.hlsl

```diff
@@ -144,10 +145,10 @@
 /// </summary>
 Varyings VertexStage(Attributes input)
 {
-    Varyings output = (Varyings)0;
+    Varyings v2f = (Varyings)0;
```

Comment:
No worries if this is a big ask, but would you mind changing v2f back to output? There are so many diffs after this change it is hard to see what has changed. Plus, other people changing this might get a lot of conflicts.

Reply:
Done.